testSrc/unit/io/flutter/android/AndroidxTest.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright  2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.android;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VirtualFile;
+import io.flutter.FlutterUtils;
+import io.flutter.module.FlutterProjectType;
+import io.flutter.testing.ProjectFixture;
+import io.flutter.testing.Testing;
+import java.io.BufferedOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.SystemIndependent;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class AndroidxTest {
+
+  private static final String ANDROID_PROPERTIES =
+    "org.gradle.jvmargs=-Xmx1536M\n";
+
+  private static final String ANDROIDX_PROPERTIES =
+    "org.gradle.jvmargs=-Xmx1536M\n" +
```

Comment:
nit: add an extra comment to the properties file and perhaps add more complex line breaking if something like
```
android.useAndroidX=
true
```
Is a valid file.

Reply:
Done.